release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -3150,3 +3206,20 @@ The components initualize successfully, and `LoadBalancer`-type services are abl
 ==== Updating
 
 To update an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+To update an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-7-46"]
+=== RHBA-2022:1018 - {product-title} 4.7.46 bug fix update
+
+Issued: 2022-03-30
+
+{product-title} release 4.7.46 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1018[RHBA-2022:1018] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1017[RHBA-2022:1017] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6844051[{product-title} 4.7.46 container image list]
+
```

Comment:
Need to add here: 

> 
[id="ocp-4-7-46-removed-feature-azure-mint-mode"]
==== Removed features

Starting with {product-title} 4.7.46, support for using the Cloud Credential Operator (CCO) in mint mode on Microsoft Azure clusters has been removed from {product-title} 4.7. This change is due to the planned link:https://azure.microsoft.com/en-us/updates/update-your-apps-to-use-microsoft-graph-before-30-june-2022/[retirement of the Azure AD Graph API by Microsoft on 30 June 2022] and is being backported to all supported versions of {product-title} in z-stream updates. For more information, see xref:../release_notes/ocp-4-7-release-notes.adoc#ocp-4-7-removed-feature-azure-mint-mode[Support for minting credentials for Microsoft Azure removed].

Reply:
It's up above under the removed features section, needs to come down here :)